config/crontab-example

```diff
@@ -37,6 +37,7 @@ MAILTO=<%= mailto %>
 45 3 * * * <%= user %> <%= vhost_dir %>/<%= vcspath %>/commonlib/bin/run-with-lockfile.sh -n <%= vhost_dir %>/stop-new-responses-on-old-requests.lock <%= vhost_dir %>/<%= vcspath %>/script/stop-new-responses-on-old-requests || echo "stalled?"
 55 4 * * * <%= user %> <%= vhost_dir %>/<%= vcspath %>/commonlib/bin/run-with-lockfile.sh -n <%= vhost_dir %>/update-public-body-stats.lock <%= vhost_dir %>/<%= vcspath %>/script/update-public-body-stats || echo "stalled?"
 0 6 * * * <%= user %> <%= vhost_dir %>/<%= vcspath %>/commonlib/bin/run-with-lockfile.sh -n <%= vhost_dir %>/send-webhook-digest.lock <%= vhost_dir %>/<%= vcspath %>/script/send-webhook-digest || echo "stalled?"
+30 4 * * * <%= user %> <%= vhost_dir %>/<%= vcspath %>/commonlib/bin/run-with-lockfile.sh -n <%= vhost_dir %>/storage-raw-emails.lock <%= vhost_dir %>/<%= vcspath %>/bin/rails storage:raw_emails:mirror storage:raw_emails:promote storage:raw_emails:unlink || echo "stalled?"
```

Comment:
Typically we create a wrapper script for rake tasks. I assume this was because of some `bundle exec` weirdness that's resolved by binstubs. Just want to double check that we know this is going to work successfully.

Reply:
Should also note this in the changelog.

Reply:
I don't think wrapper scripts are necessary any more and ideally we wouldn't use them here as once #6532 is merged this might change from `storage:raw_emails:mirror` to `storage:mirror`.

I think it might be best to document on alaveteli.org. Will link to the new page (#6796) from the changelog.